test/parallel/test-http-information-processing.js

```diff
@@ -29,24 +28,22 @@ server.listen(0, function() {
   console.error('Client sending request...');
 
   let body = '';
+  let infoCount = 0;
 
-  req.on('information', function(res) {
-    console.error('Client got 102 Processing...');
-    countdown.dec();
-  });
+  req.on('information', () => { infoCount++; });
```

Comment:
```suggestion
  req.on('information', common.mustCall(MESSAGE_COUNT));
```

and get rid of `infoCount` altogether?

Reply:
That will check that the function is called twice before exiting, but I want to preserve [the existing check](https://github.com/nodejs/node/pull/32547/files#diff-7348f0b10c259e75c1523f566e87091dL40) that it is called twice before the response is sent.

Reply:
Given that the `req.on('response')` is not wrapped in a `common.mustCall()` that check may not happen, correct? If the test fails all you'd get is a timeout.

Reply:
Sure, added common.mustCall() to the response listener as well as the end listener inside that.